ironfish-cli/src/trusted-setup/server.ts

```diff
@@ -101,7 +101,7 @@ export class CeremonyServer {
     const paramFileNames = await S3Utils.getBucketObjects(this.s3Client, this.s3Bucket)
     const validParams = paramFileNames
       .slice(0)
-      .filter((fileName) => /^params_\d{4}$/.exec(fileName)?.length === 1)
+      .filter((fileName) => /^params_\d{5}.*$/.test(fileName))
```

Comment:
is the exec -> test intentional ?

Reply:
Yeah, [Regexp.test](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp/test) returns true/false, so is more convenient for filtering than [Regexp.exec](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp/exec). They're both searching the string for a match.